deployments/deploy-ssh.sh

```diff
@@ -1,230 +1,228 @@
 #!/bin/bash -x
 
 if [ -n "${VERBOSE}" ]; then
-    VERBOSE="--v"
+  VERBOSE="--v"
```

Comment:
Do you really need these file changes in for the PR?

Reply:
Reverted the formatting